tests/chainer_tests/training_tests/extensions_tests/test_snapshot.py

```diff
@@ -97,4 +106,147 @@ def exception_raiser(trainer):
         self.assertTrue(os.path.exists(self.filename))
 
 
+@pytest.mark.parametrize('fmt', ['snapshot_iter_{}',
+                                 'snapshot_iter_{}.npz',
+                                 '{}_snapshot_man_suffix.npz'])
+def test_find_snapshot_files(fmt):
```

Comment:
Please write tests using `unittest.TestCase` for consistency.
We could allow pytest-style tests, but it should at least be consistent within a file.

Reply:
I've updated to all class-based style, though I'm not sure the usage of parameterized tests is right.

Reply:
It looks good. Thanks!